server/queue/queue-api/src/test/java/org/apache/james/queue/api/ManageableMailQueueContract.java

```diff
@@ -135,53 +135,6 @@ default void browseShouldReturnElementsInOrder() throws Exception {
             .containsExactly("name1", "name2", "name3");
     }
 
-    @Test
-    default void concurrentDequeueShouldNotAlterBrowsing() throws Exception {
```

Comment:
I think that it's still usefull to have some tests checking what is the behavior of in all this cases. For example, I'd like to make sure that I don't get an exception when flush + browse.

Reply:
Ok, but then the test is different ;-)

I'll rework this.